.infrastructure/hpa.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: todoapp
+  namespace: mateapp
```

Comment:
Ensure that the `namespace` specified (`mateapp`) exists in your Kubernetes cluster. If it doesn't, you need to create it or change this value to a valid namespace.